support-frontend/setup.sh

```diff
@@ -106,6 +106,8 @@ install_nginx() {
     EXTRA_STEPS+=("nginx is installed. Ensure you have 'include sites-enabled/*' in your nginx configuration ${NGINX_ROOT}/nginx.conf and add '127.0.0.1 support.thegulocal.com' and '127.0.0.1 support-ui.thegulocal.com' to /etc/hosts")
   fi
 
+  install_dev_nginx 
```

Comment:
Should this go [here](https://github.com/guardian/support-frontend/pull/1905/files#diff-102794ef626ea07496b9fed8cf96887aL103) instead?

Reply:
It isn't needed until nginx/setup.sh is ran